lib/ruby_indexer/lib/ruby_indexer/collector.rb

```diff
@@ -204,20 +226,26 @@ def add_constant(node, name, value = nil)
     sig { params(node: Prism::ModuleNode).void }
     def add_module_entry(node)
       name = node.constant_path.location.slice
-      return visit_child_nodes(node) unless /^[A-Z:]/.match?(name)
+      unless /^[A-Z:]/.match?(name)
```

Comment:
Not related to this PR, but is this trying to check if something is a constant? If so, this will be missing with cases where Unicode characters are being used in constant names.

For example:
```ruby
module Çava; end
defined? Çava # => "constant"
/^[A-Z:]/.match?(Çava.name) # => false
```

Reply:
I put a comment in our issue with those findings - I didn't know we wouldn't need to register all of the unicode characters, that's great.

Yeah, we were using the regex to discard usages of `module foo::Bar`, but we can indeed do something better with the node types.